Sources/TraceLog/LogLevel.swift

```diff
@@ -0,0 +1,89 @@
+///
+///  LogLevel.swift
+///
+///  Copyright 2016 Tony Stone
+///
+///  Licensed under the Apache License, Version 2.0 (the "License");
+///  you may not use this file except in compliance with the License.
+///  You may obtain a copy of the License at
+///
+///  http://www.apache.org/licenses/LICENSE-2.0
+///
+///  Unless required by applicable law or agreed to in writing, software
+///  distributed under the License is distributed on an "AS IS" BASIS,
+///  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///  See the License for the specific language governing permissions and
+///  limitations under the License.
+///
+///  Created by Tony Stone on 4/23/16.
+///
+import Swift
+import Foundation
+
+///
+/// LogLevels
+///
+/// Note: this enum must support both Swift and ObjC
+///
+@objc public enum LogLevel : Int, Comparable {
+    case off     = 0
```

Comment:
If we do want the ability to disable logs for a particular module, then having this `OFF` level is ok.

What I was suggesting is that if logging is enabled, then we do not provide the user to disable ERROR level logs, i.e., a user should never not be interested to see ERROR level logs.

The following example shows that all classes/tags will have INFO level logs, and a specific module will have TRACE4 level logs.
```
TraceLog.configure(writers: [MyWriter()], 
                                  environment: ["LOG_ALL": "INFO",
                                                        "LOG_TAG_FHA" : "TRACE4"])
```

The following example shows that all classes/tags will have ERROR (default) level logs, and a specific module will have TRACE4 level logs.
```
TraceLog.configure(writers: [MyWriter()], 
                                  environment: ["LOG_TAG_FHA" : "TRACE4"])
```

Just to clarify, the current code is correct in what it is doing. What I am suggesting is that we encourage the proper use of the logging framework, where we never lose ERROR level logs.